libs/deps/mas/countdown-timer.js

```diff
@@ -0,0 +1,124 @@
+import{LitElement as d,html as n}from"/libs/deps/lit-all.min.js";import{css as o}from"/libs/deps/lit-all.min.js";var r=o`
```

Comment:
Using lit for a simple countdown is a bit too much in my opinion. I checked the ticket and noticed the requirements to create a reusable block outside of Milo, but this request can be satisfied without adding additional dependencies. The global navigation is already available as a standalone library, it would be great to use a similar pattern to create a block that works in Milo and Dexter.

Reply:
I agree, I'll work with @rahulgupta999 to refactor it without Lit similar to our price/checkout links.

>The global navigation is already available as a standalone library, it would be great to use a similar pattern to create a block that works in Milo and Dexter.

We create the reusable features as web components but we do not package them in a single library, as it would result in the same issue w.r.t size when one of the web-component is used in the LCP section.

Reply:
Thank you, @yesil ! Make sure to check Nick's response in JIRA, it looks like reusing the component is not mandatory at this point, but it would still be great to create something that can be extended later on.

Reply:
Thanks updated the JIRA as well. Let's review once more the proposed changes are done by @rahulgupta999.

Reply:
I have updated the PR with suggestions

Reply:
Change are done. Please review